libs/ngx-dsfr/password/src/lib/password.component.ts

```diff
@@ -53,18 +58,20 @@ export class DsfrPasswordComponent implements ControlValueAccessor, OnInit, OnCh
     minDigitalCharacters: 1
   };
 
+  @Output() updatePasswordStatus = new EventEmitter<PasswordStatus>();
```

Comment:
This is the name of an event ( a reaction to something that has been done ), not a method ( something that should be done ), so I'd prefer : `passwordStatusUpdated`